src/components/Catalog/CatalogItems/CatalogItem/CatalogItem.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable no-undef */
+import React from 'react';
+import styles from './catalogItem.module.css';
+import img from '../../../../assets/img/catalog_unit_content.png';
+
+const CatalogItem = () => (
+  <div className={styles.catalogItem_item}>
+    <div className={`${styles.catalogItem_image} ibg`}>
```

Comment:
```suggestion
    <a className={`${styles.catalogItem_image} ibg`}>
```
Если я не ошибаюсь, картинка должна быть кликабельной. Надо поменять div на a

Reply:
сделано